test/framework/fileConversion.go

```diff
@@ -37,44 +40,86 @@ func FormatTestData(srcFile, tgtDir string, targetFormats ...string) (string, er
 	return srcFile, nil
 }
 
-func tarCmd(src, tgtDir string) (string, error) {
-	base := filepath.Base(src)
-	tgt := filepath.Join(tgtDir, base+image.ExtTar)
-	args := []string{"-cf", tgt, src}
+func toTar(src, tgtDir string) (string, error) {
+	tgtFile, tgtPath, err := createTargetFile(src, tgtDir, image.ExtTar)
+	defer tgtFile.Close()
 
-	if err := doCmdAndVerifyFile(tgt, "tar", args...); err != nil {
-		return "", err
+	w := tar.NewWriter(tgtFile)
+	defer w.Close()
+
+	srcFile, err := os.Open(src)
```

Comment:
something to think about (maybe) is if you want to wrap a buffer reader around this for (maybe) better performance? Depends on what buffer is done with a file reader.

Reply:
The import copy is not wrapped with a buffer reader, though I thought about it. For now this is fine as is.